### ISSUE TITLE
missing links to examples

### ISSUE THREAD

**Issue by martijnvwezel** [PointCloudLibrary/pcl#1357]
I was trying to find the correct file and this took some time, maybe it's helpfull to add the location:
pcl/doc/tutorials/content/sources/registration_api/

**Comment by jspricke** [comment]
Good point, can you send a pull request with the modification? You can find all information how to do it, here: https://github.com/PointCloudLibrary/pcl/blob/master/CONTRIBUTING.md.

**Comment by martijnvwezel** [comment]
Is it add correctly @jspricke

**Comment by jspricke** [comment]
Did you send a pull request somewhere? I didn't see it.

**Comment by zafartahirov** [comment]
Still no links to the examples?

**Comment by martijnvwezel** [comment]
I will push it tomorrow, when working.

**Comment by zafartahirov** [comment]
Hi, the examples are supposed to be here: [registration_api](https://github.com/PointCloudLibrary/pcl/tree/master/doc/tutorials/content/sources/registration_api), but there are only 2 examples there, and they are not described in the main documentation either: http://www.pointclouds.org/documentation/tutorials/registration_api.php#registration-api
